Eevee.Sleep.Bot/Utils/ConfigHelper.cs

```diff
@@ -31,6 +31,18 @@ public static class ConfigHelper {
         return GetDiscordChannelSection().GetRequiredValue<ulong>("AdminAlert");
     }
 
+    public static ulong GetJPInGameAnnouncementNoticeChannelId() {
```

Comment:
After making the nesting change of the config, make the function be `GetInGameAnnouncementNoticeChannelId(InGameAnnoucementLanguage language)` instead (so 1 function instead of 3)